src/translation/translate.js

```diff
@@ -1932,6 +1925,52 @@ Zotero.Translate.Base.prototype = {
 			: docOrElem.querySelector(selector);
 		return (elem ? elem.innerText : "").trim();
 	},
+
+	_makeSandboxItem() {
+		let sandboxZotero = this._sandboxZotero;
+		if (!sandboxZotero) {
+			throw new Error('Sandbox must be initialized');
+		}
+		
+		return class {
+			itemType;
+
+			creators = [];
+
+			notes = [];
+
+			tags = [];
+
+			seeAlso = [];
+
+			attachments = [];
+
+			constructor(itemType) {
+				this.itemType = itemType;
+			}
+
+			setExtra(field, value) {
+				this.extra = (this.extra ? this.extra + '\n' : '') + `${field}: ${value}`;
+			}
```

Comment:
My namechange from `addToExtra()` to `setExtra()` implied that we should not be adding duplicate fields, so this code should check if the field exists and if so - replace it.